handlers/templates/pages/guest-link-create.html

```diff
@@ -6,6 +6,9 @@
 
     const labelInput = document.getElementById("label");
     const expirationSelect = document.getElementById("expiration-select");
+    const expirationSelectFile = document.getElementById(
```

Comment:
Can we make this `fileExpirationSelect` and `file-expiration-select`? Otherwise, "file select" kind of sounds like it refers to a file selector element.